sources/Inputs/include/Behaviours.h

```diff
@@ -72,5 +73,43 @@ struct Generator {
   double pmax;                             ///< maximum active power
 };
 
+class HvdcLine;
+/**
+ * @brief Converter interface behaviour
+ */
+struct ConverterInterface {
+  using ConverterInterfaceId = std::string;  ///< alias for id
+  using BusId = std::string;                 ///< alias for bus id
+
+  /**
+   * @brief Constructor
+   *
+   * @param converterId the id of the converter
+   * @param busId the id of the bus
+   * @param voltageRegulationOn optional boolean for the voltage regulation parameter, only used for VSC converters
+   * @param hvdcLine the hvdc line this converter is contained into
+   */
+  explicit ConverterInterface(const ConverterInterfaceId& converterId, BusId busId, boost::optional<bool> voltageRegulationOn = {},
```

Comment:
why put default values for arguments when it is not used ?

Reply:
It is used in the networkmanager.cpp, when we create the ConverterInterface we don't know yet the value, so for readability I put the voltageRegulationOn with a default value.
So we have 
`    auto converter_1 = ConverterInterface(converter_dyn_1->getID(), converter_dyn_1->getBusInterface()->getID());
`
instead of 
`    auto converter_1 = ConverterInterface(converter_dyn_1->getID(), converter_dyn_1->getBusInterface()->getID(), boost::optional<bool>());
`